client/src/app/containers/Game/useGame.ts

```diff
@@ -0,0 +1,235 @@
+import axios from 'axios';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useAuth } from '../../../auth/useAuth';
+import { RootState } from '../../../store/store';
+import { useAlert } from '../Alert/useAlert';
+import { Card, Game, Phase } from './models';
+import {
+    actions,
+    CountCribAction,
+    CountHandAction,
+    CribAction,
+    CutAction,
+    DealAction,
+    GameAction,
+    PegAction,
+} from './slice';
+import { CreateGameResponse } from './types';
+
+interface Result {
+    game: Game;
+    selectedCards: Card[];
+    createGame: (playerIDs: string[]) => Promise<void>;
+    getGame: (id: number) => Promise<void>;
```

Comment:
:mag: :book: I am surprised that `getGame` does not return either a `Game` or a `Promise<Game>`

Reply:
Maybe a more appropriate name would be `loadGame`, as it fetches a game and loads it into redux